bin/command/command_install.js

```diff
@@ -59,86 +60,90 @@ module.exports = {
 
     installComponents()
   },
-  install: function (componentName, callback) {
-    var self = this
-    var headerRequest = { uri: repoComponents + componentName + '.json', rejectUnauthorized: false }
-    request(headerRequest, function (error, response, body) {
-      if (response && response.statusCode === 200) {
-        var componentJson = JSON.parse(body)
-        var installedMessage = component.checkInstalled(componentName)
-        if (!commands.update && installedMessage) {
-          console.log(' EXIST: '.bgBlue, 'The component ', componentName, ' has already been installed!')
-          console.log('If you need reinstall put --update in command.'.grey)
-          callback()
-          return false
-        }
-        component.install(componentJson, function (err) {
-          if (err) {
-            console.log(' ERROR: '.bgRed, 'Sorry, the component could not be installed at this time.\n', err)
+  install: async function (componentName, callback) {
+    if (await isOnline()) {
```

Comment:
MobileUI is compatible with old nodejs version (without async/await) please use callback method

Reply:
Sorry about this mistake. I already updated the code to use native module `dns` from `node` to check internet connection.